### PR TITLE
Change fmt/log to "github.com/sirupsen/logrus"

### PR DESCRIPTION
**Pull request by liwanghong**
1. import logger "github.com/sirupsen/logrus"
2. replace fmt/log with logger

### CODE REVIEW

File: core/transaction.go [hunk 1]
```diff
@@ -97,7 +98,7 @@ func (tx *Transaction) Sign(privKey ecdsa.PrivateKey, prevTXs map[string]Transac
 
 	for _, vin := range tx.Vin {
 		if prevTXs[hex.EncodeToString(vin.Txid)].ID == nil {
-			log.Panic("ERROR: Previous transaction is not correct")
+			logger.Panic("ERROR: Previous transaction is not correct")
```
Comment (yeeshi) [inline core/transaction.go]: This message is the same as line 173 but has different logger level. Plz review and use the proper one.

Comment (yeeshi) [reply core/transaction.go]: line 282 is the same case

File: core/utxo.go [hunk 2]
```diff
@@ -54,7 +54,7 @@ func deserializeUTXOIndex(d []byte) UTXOIndex {
 	decoder := gob.NewDecoder(bytes.NewReader(d))
 	err := decoder.Decode(&index)
 	if err != nil {
-		fmt.Printf("%+v\n", err.Error())
+		logger.Error(fmt.Errorf("failed to deserialize UTXOIndex: %v", err))
```
Comment (yeeshi) [inline core/utxo.go]: I think this should be panic. It's our code error which should be prevented when running in production

File: rpc/grpc_host.go [hunk 3]
```diff
@@ -56,14 +57,14 @@ func (s *Server) Start(port uint32) {
 		}
 		lis, err := net.Listen("tcp", fmt.Sprint(":",port))
 		if err != nil {
-			log.Fatalf("failed to listen: %v", err)
+			logger.Fatalf("failed to listen: %v", err)
```
Comment (yeeshi) [inline rpc/grpc_host.go]: We don't use fatalf level

File: core/transaction.go (hunk 1 above)
Comment (liwanghong) [reply core/transaction.go]: I think this must be Error，we must handle invalid data from evil node

**Pull request merged**